lib/measured/parser.rb

```diff
@@ -0,0 +1,37 @@
+module Measured::Parser
+  extend self
+
+  PARSE_REGEX = /
+    \A         # beginning of input
+    \s*        # optionally any whitespace
+    (          # capture the value
+      -?       # number can be negative
+      \d+      # must have some digits
+      (?:      # do not capture
+        [\.\/] # period or slash to split fractional part
+        \d+    # some digits after it
+      )?       # fractional part is optional
+    )
+    \s*        # optionally any space between number and unit
+    (          # capture the unit
+      [a-zA-Z] #
```

Comment:
This is because we want the unit to start with a letter, right? Maybe a comment to suggest as much?